srcpkgs/boringtun/template

```diff
@@ -0,0 +1,11 @@
+# Template file for 'boringtun'
+pkgname=boringtun
+version=vTEST2
```

Comment:
Maybe just make with 0.0.2 then and change it up in `distfiles`

Reply:
Upstream is now v0.2.0 for the tarball